sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Blob/ContainerRegistryBlobClient.cs

```diff
@@ -368,6 +373,98 @@ public virtual async Task<Response<UploadBlobResult>> UploadBlobAsync(Stream str
             }
         }
 
+        private async Task<ChunkedUploadResult> UploadInChunks(string location, Stream stream, int maxChunkSize, bool async = true, CancellationToken cancellationToken = default)
+        {
+            long length = stream.Length;
```

Comment:
Not every stream is seekable and this method can throw NotSupportedException. Do we want to build an experience around it? Not sure, if there is a specialized seekable stream type and should we consider using it.

Reply:
Thank you!  Let me look into whether we can get that value differently.